.circleci/config.yml

```diff
@@ -251,6 +270,22 @@ jobs:
       - run: cargo clippy --version
       - run: bash automation/all_clippy_checks.sh
       - save-sccache-cache
+  Generate code coverage:
+    docker:
+      - image: circleci/rust:latest
+    resource_class: large
```

Comment:
Note: this would increase our CI cost substantially (but it is way worth it)

Reply:
Looks like `large` instead of `xlarge` works...

Reply:
I've made normal tests use `large` too -- that's not really necessary to do in *this* PR, but it seems good to land the "reduce CI costs" patch at the same time as the "increase CI costs" patch.